3p/integration.js

```diff
@@ -279,17 +279,15 @@ window.draw3p = function(opt_configCallback, opt_allowed3pTypes,
     window.context.data = data;
     window.context.noContentAvailable = triggerNoContentAvailable;
     window.context.requestResize = triggerResizeRequest;
+    window.context.renderStart = triggerRenderStart;
```

Comment:
To all ad type. This is because we found that some ads other than doubleclick actually use doubleclick.js. And we want to expose this function to them